internal/cli/install/helm/install.go

```diff
@@ -77,7 +76,7 @@ func (c *Helm) Install(ctx context.Context, status *printer.StatusPrinter, opts
 			}
 
 			if !upgrade {
-				return nil, errors.New("upgrade aborted")
+				return nil, nil
```

Comment:
It'd be good to still print it:
```suggestion
				status.Step("Skipped Botkube installation upgrade.")
				status.End(true)
				return nil, nil
```